### PR TITLE
Edak added kambrook switch protocol

### PR DESCRIPTION
**Pull request by edak**
Updated protocol to be compatible with v7.0 and added to dev

### CODE REVIEW

File: libs/pilight/protocols/433.92/kambrook.c [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+        createId(id);
+        createOverallCode(unit,state);
+        createFooter();
+		kambrook->rawlen = RAW_LENGTH;
+	}
+	return EXIT_SUCCESS;
+}
+
+static void printHelp(void) {
+        printf("\t -i --id=id\tcontrol a device with this id (id) (1-29999)\n");
```
Comment (CurlyMoo) [inline libs/pilight/protocols/433.92/kambrook.c]: `(id) (1-29999)`
You can just add this information to the wiki. None of the protocols have it. Same counts for the unit.

**Comment by CurlyMoo** [comment]
Can you please check this guide and fix the PR (not reopen!) accordingly:
https://github.com/pilight/pilight/blob/master/CONTRIBUTING.md

Please also fix the alinements some tabs are actually spaces.

**Comment by pilino1234** [comment]
> If I have missed anything then I don't understand what you are asking for because it's not clear.
see
> 7. Make sure your pull-request contains one single commit.

**Comment by woutput** [comment]
My post in [this thread] (http://forum.pilight.org/Thread-How-to-combine-commits-when-using-github-online) might help you.

**Comment by wo-rasp** [comment]
Do we want to add that protocol ? If yes i would volunteer to collapse the work into one single commit.

**Pull request closed without merging**